### PR TITLE
Feature/api

### LINKED ISSUE #125: As a user, i want to create posts by using an external json API
- Key feature, requested by PO
- Authenticate via JWT

### PR DESCRIPTION
**Pull request by juanecabellob**
Closes #125

### CODE REVIEW

**Comment by Gjum** [comment]
It would be nice to have a minimal example to test this out and see how it works.

**Comment by juanecabellob** [comment]
try the /api/ url and see in the documentation how it works

**Comment by metalmatze** [comment]
Do not merge yet!

**Comment by juanecabellob** [comment]
ready to merge!

File: api/views.py [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+                        'circles': 'The int value of a circle. Default values are:\n 0 -> Me circle \n 1 -> Public circle',
+                    },
+                },
+                'Documentation': 'Please read our Manual here: https://github.com/7Pros/circuit/blob/develop/MANUAL.md',
+                'Contact': 'Please open an issue on our issue tracker: https://github.com/7Pros/circuit/issues/new',
+            },
+        })
+
+
+@api_view(['GET', 'POST'])
```
Comment (metalmatze) [inline api/views.py:75]: This says that I can make a GET request to `/api/login`. But when I do I get the following error.

download

Comment (juanecabellob) [inline api/views.py:75]: working on it

Comment (juanecabellob) [inline api/views.py:75]: fixed

**Pull request merged**